app/models/edition.rb

```diff
@@ -195,6 +195,12 @@ def self.to_date(date)
     where("editions.updated_at <= ?", date)
   end
 
+  # used by Admin::EditionFilter
+  def self.only_broken_links
+    joins("INNER JOIN link_checker_api_reports ON link_checker_api_reports.link_reportable_id = editions.id")
```

Comment:
do we need to include some sort of `where` clause against `link_reportable_type`? Sorry I've not followed the development closely, so I'm not sure what the intention of the `link_reportable_type` is, but if it ever contains anything other than `Edition`, our sql here would potentially return non edition based items that match on the id.

Reply:
Yup, this is a good point, I've added a `where` clause to deal with this